stubs/inertia/resources/js/Pages/OAuth/Partials/OAuthAppManager.vue

```diff
@@ -0,0 +1,318 @@
+        },
+    });
+};
+
+const manageOAuthApp = (app) => {
+    updateOAuthAppForm.name = app.name;
+    updateOAuthAppForm.redirect_uris = app.redirect.split(',');
+    oauthAppBeingManaged.value = app;
+};
+
+const updateOAuthApp = () => {
+    updateOAuthAppForm.put(route('oauth-apps.update', oauthAppBeingManaged.value), {
+        preserveScroll: true,
+        preserveState: true,
+        onSuccess: () => (oauthAppBeingManaged.value = null),
+    });
+};
+
+const confirmOAuthAppDeletion = (app) => {
+    oauthAppBeingDeleted.value = app;
+};
+
+const deleteOAuthApp = () => {
+    deleteOAuthAppForm.delete(route('oauth-apps.destroy', oauthAppBeingDeleted.value), {
+        preserveScroll: true,
+        preserveState: true,
+        onSuccess: () => (oauthAppBeingDeleted.value = null),
+    });
+};
+</script>
+
+<template>
+    <div>
+        <!-- Register OAuth App -->
+        <FormSection @submitted="createOAuthApp">
+            <template #title>
+                Register OAuth App
+            </template>
+
+            <template #description>
+                You may register an OAuth client to use our application's API.
+            </template>
+
+            <template #form>
+                <!-- App Name -->
+                <div class="col-span-6 sm:col-span-4">
+                    <InputLabel for="name" value="Application Name" />
+                    <TextInput
+                        id="name"
+                        v-model="createOAuthAppForm.name"
+                        type="text"
+                        class="mt-1 block w-full"
+                        autofocus
+                        autocomplete="off"
+                    />
+                    <InputError :message="createOAuthAppForm.errors.name" class="mt-2" />
+                </div>
+
+                <!-- Authorization Redirect URI -->
+                <div class="col-span-6 sm:col-span-4">
+                    <InputLabel for="redirect_uri" value="Authorization Redirect URI" />
+                    <TextInput
+                        id="redirect_uri"
+                        v-model="createOAuthAppForm.redirect_uris[0]"
+                        type="url"
+                        class="mt-1 block w-full"
+                        autocomplete="off"
+                    />
+                    <InputError :message="createOAuthAppForm.errors.redirect_uris" class="mt-2" />
+                </div>
+
+                <!-- Confidential -->
+                <div class="col-span-6 sm:col-span-4">
+                    <label for="confidential" class="flex items-center">
+                        <Checkbox id="confidential" v-model:checked="createOAuthAppForm.confidential" />
+                        <span class="ms-2 text-sm text-gray-600 dark:text-gray-400">Confidential</span>
+                    </label>
+                    <InputError :message="createOAuthAppForm.errors.confidential" class="mt-2" />
+                </div>
```

Comment:
I would love a small explanation beneath the Checkbox Label, which explains what Confidential is needed for. Similar to https://tailwindui.com/components/application-ui/forms/checkboxes#component-f03fb959d6ba814eb987d39ae40961f0

Or simply radio buttons for selecting public or confidential with a corresponding explanation.

Reply:
100%, any suggestion for the explanation?

An alternative approach is a dropdown for selecting the app type instead, like Google OAuth registration, e.g. "Web App", "Mobile and Desktop Apps", and "TVs and limited input devices":

![image](https://github.com/user-attachments/assets/ee06bf26-94d1-4fdd-b620-ebe0447dfa05)

Reply:
I would keep it simple for now, with a radio button selection of one of the two options.

I would align with https://oauth.net/2/client-types/

Confidential clients are applications that are able to securely authenticate with the authorization server, for example being able to keep their registered client secret safe.

Public clients are unable to use registered client secrets, such as applications running in a browser or on a mobile device.

Reply:
Confidential Client
Application that is able to securely authenticate with this application, without revealing the secret to an end-user.

Public Client
Unable to use registered clients secrets, such as applications running in a browser or on a mobile device.

Reply:
Thanks, I'll add this explanation for now. But Jetstream doesn't have pre-built radio button component.

We may also need a way to enable device flow for laravel/passport#1750

Github example: https://github.com/settings/apps/new

<img width="330" alt="image" src="https://github.com/user-attachments/assets/956acab4-06b2-4232-a349-bd9bcc599cdb">

Reply:
Yeah, there can be other options too.

You could use the same as the teams permissions options:
https://jetstream.laravel.com/features/teams.html

<img width="1274" alt="teams" src="https://github.com/user-attachments/assets/335271c9-7613-4475-a757-8daee15969d4">